unstructured_inference/inference/elements.py

```diff
@@ -208,18 +205,15 @@ def extract_text(
         objects: Optional[Collection[TextRegion]],
         image: Optional[Image.Image] = None,
         extract_tables: bool = False,
-        ocr_strategy: str = "auto",
-        ocr_languages: str = "eng",
     ) -> str:
         """Extracts text contained in region."""
         if self.text is not None:
             # If block text is already populated, we'll assume it's correct
             text = self.text
         elif objects is not None:
-            text = aggregate_by_block(self, image, objects, ocr_strategy)
+            text = aggregate_by_block(self, image, objects)
         elif image is not None:
-            # We don't have anything to go on but the image itself, so we use OCR
-            text = ocr(self, image, languages=ocr_languages) if ocr_strategy != "never" else ""
+            text = ""
         else:
             raise ValueError(
```

Comment:
```suggestion
            text = ""
```

Reply:
i removed the whole block of `elif image is not None:` and change the value error

Reply:
@yuming-long IMO, the `else` clause should be set to `text = ""` without raising a `ValueError`. Otherwise, image file processing will fail. 

```
    f_path = "sample-docs/layout-parser-paper-fast.jpg"

    layout = process_file_with_model(
        filename=f_path,
        model_name=None,
        is_image=True
    )
```

Reply:
ah i see, thanks!

Reply:
okay i updated this  with the remove of image parameter and looks like the output on individual blocks changed...
saw error on my unstructured branch: 
```
test_unstructured/partition/pdf-image/test_image.py::test_partition_image_hi_res_ocr_mode[individual_blocks] - AssertionError: assert 'LORE EAT O R...imeescAnelye’' == 'LayoutParser...mage Analysis'
  - LayoutParser: A Unified Toolkit for Deep Learning Based Document Image Analysis
  + LORE EAT O RES AA ERS: LER ee, A lodirininie: Hluset flocument imeescAnelye’
```
so the output is worse... i am sure why but am looking into it

Reply:
okay this is not related to this branch, the change in unstructured is causing the error